src/util/OAuth2Util.js

```diff
@@ -40,7 +40,7 @@ define(['okta', './Enums', './Errors'], function (Okta, Enums, Errors) {
     _.extend(
       oauthParams,
       _.pick(options, 'clientId', 'redirectUri'),
-      _.pick(options.authParams, 'responseType', 'responseMode', 'display', 'scope', 'scopes'),
```

Comment:
We haven't released a version that declares `scope` to be deprecated; this would be a breaking change. We should probably keep it until v2